components/03-modules/slider/slider.hbs

```diff
@@ -0,0 +1,53 @@
+<div class="slider {{ class }}" {{{ attributes }}}>
+    <div class="slider__handler">
+        <div class="slider__frame">
+            <div class="slider__slides">
+                {{#each products }}
+                    <div class="slider__item">
+                        <a href="{{ this.link }}" class="slider__link">
+                            {{ render '@image' this.image }}
+
+                            <div class="slider__item-handler">
+                                <h2 class="heading heading--fifth-level slider__name">
+                                    {{ this.name }}
+                                </h2>
+
+                                <div class="slider__price">
```

Comment:
I see that handlebars logic inside of this html tag is only about old and a new price...what about a regular price ?

Reply:
if statement added